.github/workflows/api-lint.yml

```diff
@@ -62,4 +62,5 @@ jobs:
             --types ${{ env.API_TYPES }} \
             --directories ${{ env.API_DIRECTORIES }} \
             --excludes ${{ env.API_EXCLUDES }} \
-            ${{ env.OPTION_WARNINGS }}
+            -w
```

Comment:
Please refer to the instructions at the top of the api-lint.yml GitHub Workflow file.